assets/environment.sh

```diff
@@ -1,7 +1,12 @@
 #!/bin/bash
 
+set -eu
+export DEBUG=1
```

Comment:
@AndreaCensi this should be `0` by default

Reply:
How can debug be set?